### DISCUSSION TITLE
Easier recording of portable station locations

### DISCUSSION THREAD

**Discussion by Cadair** [magicbug/Cloudlog#1081] [Ideas / Feature Requests]
My QTH isn't great for radio so I tend to mostly go out portable. At the moment if I want to accurately record my location I need to fill in a rather large form for a new station location. I can think of two things that would help this, the first (easy) one is to be able to copy a station location and edit it. Then I could setup one /P and then just copy it and move my locator.

Ideally though, it would be nice to be able to configure a /P station and then put in the locator for each QSO (or enter it on the live QSO page).

**Comment by magicbug** [comment]
The current fields for "Station Locations" really is the minimum needed for most of the common awards, however some of the fields could be automatically populated for example based on the callsign and using javascript and geo location tools to do gridsquares.

Certainly don't want to remove features that would present other issues, but happy to look at some auto population or accept a PR if anyone fancies a go

**Reply by Cadair** [reply]
Yeah, I get most of them are useful, but the only one I am ever really changing when I go portable is my locator square the rest I would like to be able to copy.

**Reply by magicbug** [reply]
Copy feature seems good!

**Reply by magicbug** [reply]
Added to the todo list! https://github.com/magicbug/Cloudlog/projects/8

**Comment by Cadair** [comment]
Hello again,

I have done a bunch of portable ops this weekend and I have found that there seems to be no way to get all the stats (and awards?) combined over all the station locations?

Thanks!
